src/macrofier/types.mjs

```diff
@@ -644,4 +685,4 @@ export default {
   getSchemaType,
   getJsonType,
```

Comment:
getJSONType wasn't really being used before. I see that you're using it to do some local work in this method, which is cool, but we should stop exporting it now that you've got logic there.

Don't want any external dependencies on it.

Reply:
removed it